### ISSUE TITLE
Not possible to print to stdout/stderr by listeners or otherwise inside `Run Keyword` variants

### ISSUE THREAD

**Issue by menonprasad1** [robotframework/robotframework#2062]
Description :

 I'm trying to capture start_keyword event in the listener. It is observed that if the keword is explicitly called then the start_keyword even is called for that keyword and any other keywords that this keyword relies upon. However if the keyword name is assigned to a variable and the keyword is called using 'Run Keyword', then only the start_keyword for 'Run Keyword' is called and no the underlying keywords.

**Comment by pekkaklarck** [comment]
I cannot reproduce this. `start_keyword` is called just fine also with keywords wrapped with `Run Keyword`.

I noticed, however, that messages printed by listeners to the stdout inside `Run Keyword` are not visible on the console. Is this also what you are seeing? This problem is caused by Robot capturing stdout and stderr when it is running library keywords. It seems to be pretty easy to fix by special casing `Run Keyword` and other similar keywords so that outputs are not captured.

A workaround for the problem I encountered is not using `print message` but instead `print >> sys.__stdout__, message` or `sys.__stdout__.write(message + '\n')`.
